app-directory/i18n/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules\\typescript\\lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
```

Comment:
What does it do?

Reply:
Its added automatically for the next typescript plugin